packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -34,22 +34,29 @@ type HookProps = {
   environment: TApplicationContext<{}>['environment'];
   project: TFetchProjectQuery['project'];
 };
+type TMousePosition = {
+  clientX: number;
+  clientY: number;
+};
 type State = {
   activeItemIndex?: string;
   isExpanderVisible: boolean;
   isMenuOpen: boolean;
+  mousePosition: TMousePosition;
 };
 type Action =
   | { type: 'setActiveItemIndex'; payload: string }
   | { type: 'unsetActiveItemIndex' }
   | { type: 'setIsExpanderVisible' }
   | { type: 'toggleIsMenuOpen' }
   | { type: 'setIsMenuOpenAndMakeExpanderVisible'; payload: boolean }
+  | { type: 'setMousePosition'; payload: TMousePosition }
```

Comment:
nit: I suppose we read the mouse position, so maybe `getMousePosition` would be more suitable

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3536/commits/b662e8f9258af8523279fa2a301a35959e6a6b33